Themis.xcodeproj/project.pbxproj

```diff
@@ -1497,6 +1439,7 @@
 					"@executable_path/../Frameworks",
 					"@loader_path/Frameworks",
 				);
+				MACOSX_DEPLOYMENT_TARGET = 10.13;
```

Comment:
Docs currently say we support macOS 10.12+

Not really against dropping support, but again, I don't think it should happen in a patch release.

Though, given that we've lost ability to test for it, maybe it's warranted in this case.

Reply:
The message from XCodeBuild about minimum version 10.13 is still a warning, not an error. I may be getting ahead. I just looked for the error and cleared the output from the warnings.